lib/lambdalocal.js

```diff
@@ -6,81 +6,76 @@
  * https://docs.aws.amazon.com/en_us/lambda/latest/dg/nodejs-prog-model-context.html
  */
 
-const mute = require('mute'),
-  utils = require('./utils.js'),
-  _context = require('./context.js');
+var logger = require('winston');
 
+const mute = require('mute'),
+    utils = require('./utils.js'),
+    _context = require('./context.js');
+	
+var _setLogger = function(_logger){
+	logger = _logger;
+}
+    
 var _execute = function(opts) {
-  var event = opts.event,
-    lambdaPath = opts.lambdaPath,
-    lambdaHandler = opts.lambdaHandler || 'handler',
-    profilePath = opts.profilePath,
-    profileName = opts.profileName,
-    region = opts.region || 'us-east-1',
-    callbackWaitsForEmptyEventLoop = (typeof opts.callbackWaitsForEmptyEventLoop === 'undefined') ?
-      true : opts.callbackWaitsForEmptyEventLoop,
-    timeoutMs = (typeof opts.timeoutMs === 'undefined') ? 3000 : opts.timeoutMs,
-    muteLogs = opts.mute,
-    unmute = null,
-    callback = opts.callback;
+    var event = opts.event,
+        lambdaPath = opts.lambdaPath,
+        lambdaHandler = opts.lambdaHandler || 'handler',
+        profilePath = opts.profilePath,
+        profileName = opts.profileName,
+        region = opts.region || 'us-east-1',
+        callbackWaitsForEmptyEventLoop = (typeof opts.callbackWaitsForEmptyEventLoop === 'undefined') ?
+            true : opts.callbackWaitsForEmptyEventLoop,
+        timeoutMs = (typeof opts.timeoutMs === 'undefined') ? 3000 : opts.timeoutMs,
+		muteLogs = opts.mute,
```

Comment:
All of thoose indents do not appear for me..... I'll modify it directly in github